lib/server/connect.js

```diff
@@ -32,6 +54,18 @@ Meteor.methods({
     check(browserId, String);
     check(clientId, String);
     check(data, Object);
+    
+    if(AppConfig.env === 'production') {
+      // get session id from kadiraInfo Object
+      var kadiraInfo = Kadira._getInfo();
+      var sessionId = kadiraInfo.session;
+
+      if(!AppConfig._authorizedClientSessions[sessionId]) {
+        // not a valid session
+        // return without proceed
+        return;
```

Comment:
Same as the previous comment.
See: https://github.com/kadirahq/meteor-debug/pull/20/files#r43823648